src/solvers/dg_unstructured_quad/sort_boundary_conditions.jl

```diff
@@ -20,6 +20,21 @@ end
 # from the `UnstructuredBoundaryContainer2D` in cache.boundaries according to the boundary types
 # and stores the associated global boundary indexing in NTuple
 function UnstructuredQuadSortedBoundaryTypes(boundary_conditions::Dict, cache)
```

Comment:
This needs to get another name and another home since it's used for more than one mesh type. Could you please add it to one of the TODO list?

Reply:
I think the point "Move stuff to appropriate files (definition of DG to `dg_common.jl` etc.)" in #542 requires all these files to be revised anyway. I didn't want to move anything in this PR because that makes it much harder to see what actually changed when reviewing the PR.
I also used the `UnstructuredQuadMesh` BC stuff, which should be moved to common too. Oh, and the volume integral from `CurvedMesh`.

Reply:
Okay :+1: